### PR TITLE
New setup

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

**Comment by John-Colvin** [comment]
fixes https://github.com/DlangScience/DlangScience.github.io/issues/4

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+---
```
Comment (9il) [inline index.html:3]: Looks like markdown problems.

Comment (John-Colvin) [inline index.html:3]: Nope. That's YAML instructions for Jekyll.

Comment (9il) [inline index.html:3]: This should be in html?

Comment (John-Colvin) [inline index.html:3]: Yes. The html files here are processed by Jekyll on github's servers before being served

**Comment by John-Colvin** [comment]
thanks :)

**Pull request merged**